python/TestHarness/testers/RunApp.py

```diff
@@ -208,6 +208,10 @@ def getPBSCommand(self, options):
     if options.PBSEmulator:
       self.specs['pbs_stdout'] = 'pbs_stdout = PBS_EMULATOR'
       self.specs['pbs_stderr'] = 'pbs_stderr = PBS_EMULATOR'
+    else:
```

Comment:
I was confused as to why this was the only option added to the "else" clause since it should be fine to add a project to the emulator (unecessary, but fine). However, you indicated that it does indeed create an error to pass it to the emulator. That's important information for posterity. Would you mind explicitly mentioning that here in a comment?